src/NuGet.Services.EndToEnd/Support/PackageType.cs

```diff
@@ -19,5 +19,6 @@ public enum PackageType
         SemVer2StableMetadataUnlisted,
         FullValidation,
         Signed,
+        SymbolsPackage
```

Comment:
nit: trailing comma

Reply:
not much of a fan of ending in comma, but don't feel too strongly.. added.

Reply:
it is why this diff is very clear 😄